bin/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+# Tag, Push and Deploy only if it's not a pull request
+if [ -z "$TRAVIS_PULL_REQUEST" ] || [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+
+    # Push only if we're testing the staging branch
+    if [ "$TRAVIS_BRANCH" == "staging" ]; then
```

Comment:
Based on discussion & comments with Brian & Nathan, let's change this to "master" from "staging"

Reply:
OK, will add another commit